src/Toggle/ToggleChecker.php

```diff
@@ -51,9 +46,28 @@ public function isToggleActive( $toggleName )
         if ( !$this->toggleGateway->isToggleVisibleForUsers($toggleName) ) {
             return false;
         }
-        $groupPolicyResponse = $this->groupPolicy->getTogglePolicy($toggleName, $this->group);
+        $groupPolicyResponse = $this->transformByType(
+            $this->groupPolicy->getTogglePolicy($toggleName, $this->group),
+            $this->toggleGateway->getType($toggleName));
         return $groupPolicyResponse->isEnabled() ||
```

Comment:
According to this, every time if a toggle is disabled in a group policy, there's no way for users to enable it even if it's not a 'group' type toggle.